interactions/api/models/message.py

```diff
@@ -189,7 +189,7 @@ class Message(DictSerializerMixin):
     :ivar Optional[MessageInteraction] interaction?: Message interaction object, if the message is sent by an interaction.
     :ivar Optional[Channel] thread:? The thread that started from this message, if any, with a thread member object embedded.
     :ivar Optional[Union[Component, List[Component]]] components?: Components associated with this message, if any.
-    :ivar Optional[List[PartialSticker"]] sticker_items?: An array of message sticker item objects, if sent with them.
+    :ivar Optional[List["PartialSticker"]] sticker_items?: An array of message sticker item objects, if sent with them.
```

Comment:
```suggestion
    :ivar Optional[List[PartialSticker]] sticker_items?: An array of message sticker item objects, if sent with them.
```

This was probably a mistake when I was doing the documentation. I took a lazy approach to copy-pasting the interfacing which introduced the quotations.